mage/dependency.go

```diff
@@ -44,15 +44,14 @@ func EnsureZeitgeist(version string) error {
 	}
 
 	if _, err := semver.ParseTolerant(version); err != nil {
-		return errors.Wrapf(
-			err,
-			"%s was not SemVer-compliant. Cannot continue.",
-			version,
+		return fmt.Errorf(
+			"%s was not SemVer-compliant. Cannot continue.: %w",
+			version, err,
```

Comment:
Nit: should we improve the formatting of this error? `Cannot continue.:` looks sort of strange.

Reply:
Yep, changed to improve readability.